packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaManager.ts

```diff
@@ -663,6 +663,56 @@ describe('RemoteMediaManager', () => {
       remoteMediaManager.stop();
     });
   });
+
+  describe('setPreferLiveVideo', () => {
+
+    it('sets preferLiveVideo to true', async () => {
+      const config = cloneDeep(DefaultTestConfiguration);
+
+      config.video.initialLayoutId = 'OnePlusFive';
+
+      remoteMediaManager = new RemoteMediaManager(
+        fakeReceiveSlotManager,
+        fakeMediaRequestManagers,
+        config
+      );
+  
+      await remoteMediaManager.start();
+  
+      resetHistory();
+      remoteMediaManager.setPreferLiveVideo(true);
+
+      expect(config.video.preferLiveVideo).to.equal(true);
+
+      Object.values(remoteMediaManager.media.video.activeSpeakerGroups).forEach((activeSpeakerGroup: any) => {
+        expect(activeSpeakerGroup.options.preferLiveVideo).to.equal(true);
```

Comment:
it would be better to check that RemoteMediaGroup.setPreferLiveVideo() was called rather than accessing private options of RemoteMediaGroup, because that's an implementation detail of RemoteMediaGroup that RemoteMediaManager and its tests should not know about